liquidprompt

```diff
@@ -1064,6 +1234,26 @@ _lp_fill() {
     lp_fill="${left}${filled}${right}"
 }
 
+_lp_substitute() { # string pairs_array
+    # Replace string with the "right" part of the first pair in the given array, if the "left" part matches the string.
+    # FIXME If we ever support Bash>4.3, this would be better implemented with associative arrays.
+    lp_substitute=""
+    local string="${1}"
+    shift 1
+    local pairs=("$@")
```

Comment:
This breaks on Zsh 5.0
```suggestion
    local pairs
    pairs=("$@")
```

Reply:
Spotted another one line 260 `local active=()` — all fixed.